data/data/aws/vpc/sg-master.tf

```diff
@@ -31,9 +31,9 @@ resource "aws_security_group_rule" "master_ingress_icmp" {
   security_group_id = "${aws_security_group.master.id}"
 
   protocol    = "icmp"
-  cidr_blocks = ["${data.aws_vpc.cluster_vpc.cidr_block}"]
-  from_port   = 0
-  to_port     = 0
+  cidr_blocks = ["0.0.0.0/0"]
+  from_port   = -1
+  to_port     = -1
```

Comment:
[The docs][1] for these have>

> ... or ICMP type number if protocol is "icmp"...

From [the IANA registry][2], it looks like we'd want to allow 8 (Echo) and 0 (Echo Reply).  Do we want to block or allow the other types?  Personally, I'm in favor of necking this down to the absolute minimum and then opening back up individual types as we have requests for them (e.g. "we want ping support, and that requires types 0 and 8").

[1]: https://www.terraform.io/docs/providers/aws/r/security_group_rule.html#from_port
[2]: https://www.iana.org/assignments/icmp-parameters/icmp-parameters.xhtml